variables.tf

```diff
@@ -1,3 +1,15 @@
+variable "alarm_suffix" {
+  type        = string
+  default     = ""
+  description = "Suffix to add to alarm name, used for separating different AWS account."
+}
+
+variable "govcloud_check" {
```

Comment:
One suggestion and this may be to fancy. What if we just detect if it's a govcloud environment based of the data partition value https://www.terraform.io/docs/providers/aws/d/partition.html that way it would "just work" depending on where the module is deployed.

Reply:
This seems like a good idea to me. It might be as simple as:

```hcl
event_pattern = data.aws_partition.current.partition == "aws-us-gov" ? file("${path.module}/govcloud-event-pattern.json") : file("${path.module}/event-pattern.json")
```

Reply:
great idea